packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -2017,24 +1994,7 @@ export class Grapher
         )
             return this.selectedFacetStrategy
 
-        if (this.disableAutoFaceting) return FacetStrategy.none
-
-        // Auto facet on SingleEntity charts with multiple selected entities
-        if (
-            this.addCountryMode === EntitySelectionMode.SingleEntity &&
-            this.selection.numSelectedEntities > 1
-        )
-            return FacetStrategy.entity
-
-        // Auto facet when multiple slugs and multiple entities selected. todo: not sure if this is correct.
-        if (
-            this.addCountryMode === EntitySelectionMode.MultipleEntities &&
-            this.hasMultipleYColumns &&
-            this.selection.numSelectedEntities > 1
-        )
-            return FacetStrategy.metric
```

Comment:
This is the code I refer to in my _outstanding question_:
> We had some code for auto faceting, which was a disabled feature until now.
      In some of the cases that it had we essentially auto-facet now (the cases where _only_ a faceted chart makes sense), but the feature also covers cases where faceting would be nice-to-have.
      I removed it, should we keep it instead?

I think the code here would, for example, also auto-facet multi-entity, multi-metric line charts, which we currently don't do.